deepforest/utilities.py

```diff
@@ -388,6 +388,14 @@ def boxes_to_shapefile(df, root_dir, projected=True, flip_y_axis=False):
     Returns:
        df: a geospatial dataframe with the boxes optionally transformed to the target crs
     """
+    # Check if the user has set flip_y_axis, but not projected. Warn and confirm.
```

Comment:
Change this comment to 

`# raise a warning if user sets projected to be True and flip_y_axis is True`

and remove the () on the if statement
 
`if flip_y_axis and projected:`